website/docs/reference/supported_commands.md

```diff
@@ -108,6 +108,17 @@ The following operators and modifiers are available in the `update` and `findAnd
 |                   | `$sort`     | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/831)  |
 |                   | `$bit`      | ⚠️     | [Issue](https://github.com/FerretDB/FerretDB/issues/821)  |
 
+### Projection Operators
+
+The following operators are available in the `find` command `projection` argument.
+
+| Operator     | Status | Comments       |
+|--------------|--------|----------------|
+| `$`          | ❌      | Unimplemented  |
+| `$elemMatch` | ❌      | Unimplemented  |
+| `$meta`      | ❌      | Unimplemented  |
+| `$slice`     | ❌      | Unimplemented  |
```

Comment:
Let's add issues links (and create them if needed)

Reply:
Added :+1: